packages/waku/src/lib/middleware/rsc/ssr.ts

```diff
@@ -62,22 +62,29 @@ export const shutdown = async () => {
 export const loadServerFile = async (
   fname: string,
   command: 'dev' | 'build' | 'start',
+  isModule: boolean,
 ) => {
   if (command !== 'dev') {
-    return import(fname);
+    if (isModule) {
+      return import(fname);
+    }
+    return import(url.pathToFileURL(fname).toString());
   }
   const vite = await getViteServer();
+  if (fname.startsWith('file://')) {
+    fname = url.fileURLToPath(fname);
+  }
   return vite.ssrLoadModule(fname);
 };
 
 // FIXME this is very hacky
 const createTranspiler = async (cleanupFns: Set<() => void>) => {
-  return (file: string, name: string) => {
+  return (file: URL, name: string) => {
```

Comment:
Not a big fan of receiving an URL object and the implicit `toString()`.
```suggestion
  return (fileURL: string, name: string) => {
```